spec/cirro_io_v2/resources/gig_result_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe CirroIOV2::Resources::GigResult do
+  let(:site) { 'http://api.cirro.io' }
+  let(:client) do
+    CirroIOV2::Client.new(private_key: File.read('./spec/fixtures/private_key.pem'),
+                          client_id: 1,
+                          site: site)
+  end
+
+  describe '#create' do
+    let(:params) do
+      {
+        gig_task_id: '1',
+        user_id: '1',
+        title: 'Batman Begins',
+        description: "All right, Mr. DeMille, I'm ready for my closeup.",
+        quantity: 2,
+        multiplier: '1.0',
+        delivery_date: '2022-06-13',
+        cost_center_data: {
+          company: 'Schinner, Brakus and Ortiz',
+          inhouse: 'no',
+          customer_id: '1234',
+          legal_entity: 'GmbH',
+        },
+      }
+    end
+
+    it 'creates a gig time activity' do
```

Comment:
:D there's a lot copy pasting going on here apparently
it's a gig result not a gig time activity